README.md

```diff
@@ -51,33 +51,6 @@ For IntelliJ IDEA users, please refer to [this](https://github.com/intellij-rust
 
 In order to build `tonic` >= 0.8.0, you need the `protoc` Protocol Buffers compiler, along with Protocol Buffers resource files.
 
-#### Ubuntu
```

Comment:
What's the motivation for this? Are there clear instructions (like these) on any of the linked websites on how to install?

Reply:
> What's the motivation for this?

I feel it is outside the scope of this repository to provide installation instructions for `protoc` on a wide variety of operating systems.

> Are there clear instructions (like these) on any of the linked websites on how to install?

I couldn't find them.

Reply:
> > What's the motivation for this?
> 
> I feel it is outside the scope of this repository to provide installation instructions for `protoc` on a wide variety of operating systems.

Not sure I agree, since it is kind of important part of building stuff on tonic, the need for external tooling is something Rust folks are typically not used to, I think it's useful to our users to keep some instructions around. (Until we migrate to using protox by default, I suppose.)

Reply:
`prost-build` seems to print the guide for `mac`, `debian` and `ubuntu`. Are not these sufficient?

https://github.com/tokio-rs/prost/blob/9ed944eb633480079037dfceeee61aac6cd0c94f/prost-build/src/config.rs#L1207-L1224

Reply:
I guess.